.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question
+about: Who, what, when, where, and how?
+
+---
+
+Hey, stop right there!
+
+We use GitHub to track feature requests and bug reports. Please do not submit issues for questions about how to use JanusGraph, how to set JanusGraph up, best practices, or development related help.
+
+However, we do want to help! Head on over to our mailing lists and ask
+your questions there. In additional to awesome, knowledgeable community
+contributors, core JanusGraph developers check the mailing list.
+
+Mailing list for usage, setup and best practices is https://groups.google.com/forum/#!forum/janusgraph-users/join.
```

Comment:
I think we should also include `troubleshooting` here. I.e.:

`Mailing list for troubleshooting, usage, setup and best practices...`